lib/susedistribution.pm

```diff
@@ -161,9 +161,9 @@ sub init_desktop_runner {
     # https://progress.opensuse.org/issues/35589
     if (check_var('DESKTOP', 'kde')) {
         if (get_var('WAYLAND')) {
-            wait_still_screen(3);
-            type_string_very_slow substr $program, 0, 2;
-            wait_still_screen(3);
+            wait_still_screen(2);
+            type_string_very_slow substr $program, 0, 1;
+            type_string_very_slow substr $program, 1, 1;
             type_string_very_slow substr $program, 2;
```

Comment:
@Vogtinator the "typing character by character" part is already in here. I will try to push it further by checking if the desktop runner is there after every single character on wayland.

Reply:
I don't think that would mitigate the issue - it would only detect the failure earlier.